src/test/java/io/confluent/examples/streams/CustomStreamTableJoinIntegrationTest.java

```diff
@@ -319,14 +320,14 @@ public void shouldTriggerStreamTableJoinFromTable() throws Exception {
       return new Transformer<String, Double, KeyValue<String, Pair<Double, Long>>>() {
 
         private KeyValueStore<String, Pair<Double, Instant>> streamBufferStore;
-        private KeyValueStore<String, Long> tableStore;
+        private KeyValueStore<String, ValueAndTimestamp<Long>> tableStore;
         private ProcessorContext context;
 
         @SuppressWarnings("unchecked")
         @Override
         public void init(final ProcessorContext context) {
           streamBufferStore = (KeyValueStore<String, Pair<Double, Instant>>) context.getStateStore(streamBufferStoreName);
-          tableStore = (KeyValueStore<String, Long>) context.getStateStore(tableStoreName);
+          tableStore = (KeyValueStore<String, ValueAndTimestamp<Long>>) context.getStateStore(tableStoreName);
```

Comment:
I'm not too comfortable with this API breakage, but it seems like it's too late to do anything about it now...

Reply:
Well. Thinking about it, we might not want to recommend this pattern in the first place... \cc @miguno 

It's actually quite a hack to access to store of a DSL operator in a custom transformer from my point of view... Thoughts?